rpcs3/CMakeLists.txt

```diff
@@ -153,7 +153,9 @@ elseif(UNIX)
             ${CMAKE_SOURCE_DIR}/bin/git $<TARGET_FILE_DIR:rpcs3>/git)
 elseif(WIN32)
     add_custom_command(TARGET rpcs3 POST_BUILD
+        COMMAND "${CMAKE_COMMAND}" -E remove "${CMAKE_SOURCE_DIR}/bin/rpcs3.exe"
```

Comment:
Why is this necessary?

Reply:
because the post build process replaced the compiled `rpcs3.exe` with the one present in `rpcs3/bin`.
I would prefer something like `EXCLUDE rpcs3.exe` in the next command (with the command `-E copy_directory`) but I didn't find any option on that command.